### ISSUE TITLE
Can't see CsvReader.getRecords<T>() method?

### ISSUE THREAD

**Issue by KMelendez** [JoshClose/CsvHelper#377]
I've installed CsvHelper via NuGet in VS2012 and LyncPad 4. However I can't use the getRecords() method in my code.

I noticed that this issue has been posted before, apparently the assembly references the wrong folder due to some bug in NuGet. How can I change this manually both in my project and in LyncPad? Is it something that VS2012 did? I had LyncPad running CsvHelper just fine yesterday, but I re-installed it to troubleshoot the problem I had in VS2012, and now both are broken.

**Comment by JoshClose** [comment]
When you say "LyncPad" are you referring to https://www.linqpad.net/? Are you having trouble using it in LinqPad? I have used the latest version (2.13.0) in LinqPad and it worked just fine for me.

If you are having trouble getting it to work through VS2012, what type of project is it? What version of the .NET framework are you targeting?

**Comment by KMelendez** [comment]
Yes, sorry I mispelled that. I managed to fix it. It's the same issue as this one:

https://github.com/JoshClose/CsvHelper/issues/49

I was referring that solution, in which it was suggested to remove and manually place a reference to the correct assembly. I managed to that now. Has the bug in NuGet been addressed yet? As far as I can tell I have the latest version.

Turns out the one in Lynqpad was just my mistake. The auto-complete was zoning out for a moment and I misspelled the function.
